<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowForm</class>
 <widget class="QWidget" name="ShowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QTableView" name="viewLayer1"/>
   </item>
   <item row="0" column="1">
    <widget class="QTableView" name="viewLayer0"/>
   </item>
   <item row="1" column="1">
    <widget class="QTableView" name="viewFinal"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
